### ISSUE TITLE
In translate.py 228-230

### ISSUE THREAD

**Issue by WoNiuHu** [BladeSun/seq2sql#2]
hi , in translate.py , you change the learning_rate by loss , I think it's a good idea, can this support distribute train? I'm also thinking about using loss to control  termination,  but did not get a good method in   distribute train.

**Comment by BladeSun** [comment]
This code is a copy from https://github.com/tensorflow/models/tree/master/tutorials/rnn/translate, details of this code can be found at https://www.tensorflow.org/tutorials/seq2seq. Currently, I'm modifying this code to implement a seq2sql model for WikiSQL dataset. And it is not finished.
